src/modules/data/Hash.ts

```diff
@@ -42,8 +43,11 @@ export class Hash
     constructor (bin?: Buffer)
     {
         this.data = Buffer.alloc(Hash.Width);
-        if (bin != undefined)
+        if (bin !== undefined)
+        {
+            assert.strictEqual(bin.length, Hash.Width);
```

Comment:
How does It handle the error here?

Reply:
The following error is thrown:
AssertionError [ERR_ASSERTION]: Expected values to be strictly equal: